governance/polls/poll-parameters/goerli-approval-sentiment-example.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Approval voting sentiment example
+summary: Signal your support for your favourite Core Unit(s)
+discussion_link: https://forum.makerdao.com/t/signal-request-set-psm-fees-to-0/10894
+parameters:
+  input_format:
+    type: choose-free
+    options: [1,2,3,4]
+    abstain: [0]
+  victory_conditions:
+    - { type : approval, options: [1,2,3,4] }
```

Comment:
if you want option 4 not to be able to be selected at the same time than option 1, 2, you should put "options": [1,2,3]

Reply:
Yeah for this scenario it makes sense to have 4 separate, will update it

Reply:
Just to make absolutely sure though, if option 4 got the highest amount of approval, it would still be the winner? Even if taken out of the victory conditions section?